homeassistant/components/daikin/sensor.py

```diff
@@ -80,7 +81,7 @@ class DaikinSensorEntityDescription(SensorEntityDescription, DaikinRequiredKeysM
     ),
     DaikinSensorEntityDescription(
         key=ATTR_TOTAL_POWER,
-        name="Total Power Consumption",
+        name="Current Total Power Consumption",
```

Comment:
Hi, you said in https://bitbucket.org/mustang51/pydaikin/pull-requests/47#comment-262256644, it would be less misleading to explicitly call this sensor "Estimated" or "Computed"

Reply:
Yes true should we add it to the name in home assistant or is it enough to add it to the home assistant documentation? (https://github.com/home-assistant/home-assistant.io/pull/20749)

Reply:
I added an explanation to the documentation...  I still would like to remove the "total" and rather call it "Estimated Power Consumption" (even drop the "current" since its not that "current" because of the calculation)
do you agree?